WooCommerce/Classes/ViewRelated/Dashboard/Factories/DashboardTopBannerFactory.swift

```diff
@@ -0,0 +1,50 @@
+import Gridicons
+import UIKit
+
+/// Generates top banner view that is shown when the eligible stats version is from v3 to v4 and v4 to v3.
```

Comment:
Would I make sense to generalise the comment to just mention that it's shown at the top of the Dashboard? It looks like that we might use it to present other relevant info?